src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -174,6 +195,13 @@ Plugin::Plugin() {
     for (const auto& device : m_device_map) {
         m_configs_map.insert({device.first, ExecutionConfig(ov::device::id(device.first))});
     }
+
+    // Set runtime info
+    auto& ov_version = ov::get_openvino_version();
+    m_compiled_model_format["OV_VERSION"] = ov_version.buildNumber;
+    for (const auto& device : m_device_map) {
+        m_compiled_model_format[device.first] = device.second->get_info().driver_version;
```

Comment:
I suppose we need to take driver version specific to device we are compile_model on.
But I hope that a driver is the same for all devices and we can collect only driver of first device here (otherwise depending on actual number of devices on the system, we can have different `m_compiled_model_format`)

@isanghao could you please confirm?

Reply:
We print the GPU info in one iGPU + dGPU machine:

```
    for (const auto& device : m_device_map) {
        auto driver_info = device.second->get_info();
        std::cout << device.first << ": (" << driver_info.dev_name << ", " << driver_info.vendor_id << ", "
                  << driver_info.device_id << ", " << driver_info.driver_version << ")" << std::endl;
    }
```

```
0: (Intel(R) UHD Graphics 770,         32902, 42880, 23.17.26241.33)
1: (Intel(R) Arc(TM) A770 Graphics, 32902, 22176, 23.17.26241.33)
2: (Intel(R) Arc(TM) A770 Graphics, 32902, 22176, 23.17.26241.33)
```

It seems iGPU and dGPU have the same driver version, I'm not sure whether they can be different driver version in the same machine.

Reply:
I suppose driver version is defined by `driver_info.driver_version` which is the same for all devices, while other values are per-device properties. I hope that device_id and vendor_id are driver independent.

CC @isanghao

Reply:
@riverlijunjie I suppose we need to store driver version for a single device, because depending on a number of devices on the system we will have different value for `m_compiled_model_format`

Reply:
Got it, will do it!